### ISSUE TITLE
Make SkeletonClipping work

### ISSUE THREAD

**Issue by jeremyfa** [jeremyfa/spine-hx#7]
SkeletonClipping doesn't seem to work as expected. That's not a problem as long as we don't use clipping, but as we seek parity with original spine runtime, we should investigate what makes clipping fail.

Best strategy so far to find the issue is to run the same animation (with clipping) on both spine-libgdx and spine-hx and see where the values get different.

**Comment by rainyt** [comment]
Hi,Is there any good news

**Comment by jeremyfa** [comment]
Nope! Not my priority to make it work at the moment unfortunately. Maybe SkeletonClipping code is fine, I just didn’t manage to make it work in the renderer I am using.

**Comment by rainyt** [comment]
https://github.com/rainyt/openfl-spine/blob/master/src/spine/openfl/SkeletonSprite.hx

I've added mask support for openfl-spine and haven't modified spine-hx, so SkeletonClipping should work properly.  I think rendering failure may be caused by arrays.

**Comment by jeremyfa** [comment]
That’s good to know, thanks for the feedback!
